airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -112,7 +112,7 @@ definitions:
           - ERROR
       emitted_at:
         description: "the time in ms that the message was emitted"
-        type: number
+        type: integer
```

Comment:
Is it store somewhere? I wonder if this could generate serialization issues.

Reply:
I'm not sure what you mean exactly. `number` is actually a double, which we don't want for a timestamp so changing it to `integer` makes it a long, like the other timestamps.

Reply:
I am wondering is this change can cause a serialization issue because of some trace message are serialized using a double.

Reply:
I don't think anything is currently producing or storing trace messages anywhere yet, so AFAIK this should be a pretty safe change to make right now

Reply:
fyi: trace messages have been implemented as Doubles in Java & Python to match the original spec. We've also version-bumped GA connectors to start emitting trace messages so we may already be producing them with `emitted_at` as Double